src/main/resources/swagger/api-docs.yaml

```diff
@@ -5921,6 +5921,9 @@ definitions:
       enableFlowLogs:
         type: boolean
         description: Optional, false if not specified. If true, enable flow logs within the high security network. Requires highSecurityNetwork = true.
+      privateIpGoogleAccess:
+        type: boolean
+        description: Optional, false if not specified. If true, it configures the VPC network to only allow access to GCP APIs that are protected by the project's service perimeter and routes all allowed API traffic through a narrow IP range. Requires highSecurityNetwork = true.
```

Comment:
`Optional, false if not specified.  If set to true, also requires HighSecurityNetwork=true`. Basically just move the High Security Network requirement up to the front because it's an important little gotcha that might get overlooked if it's at the end.  People hate reading.